projects/smart-table/src/lib/smart-table/smart-table.component.html

```diff
@@ -1,93 +1,101 @@
-            </ng-container>
-        </ng-container>
-        <div class="aui-smart-table-filter a-input" *ngIf="optionalFilters.length">
-            <label class="a-input__label">&nbsp;</label>
-            <div class="a-input__wrapper">
-                <button class="a-button-outline has-icon-left" (click)="toggleOptionalFilters()">
-                    <span class="fa fa-filter"></span> {{ options.translations?.moreFilters || 'Meer filters'}}
-                </button>
-            </div>
-        </div>
-        <div class="aui-smart-table-filter a-input" *ngIf="totalResults > 0">
-            <label class="a-input__label">&nbsp;</label>
-            <div class="a-input__wrapper">
-                <button class="a-button-outline has-icon-left" (click)="exportToExcel()">
-                    <span class="fa fa-file-excel-o"></span> {{ options.translations?.export || 'Export'}}
-                </button>
+  <div class="aui-smart-table-filters">
+    <aui-table-input-filter class="aui-smart-table-filter" *ngIf="genericFilter" [filter]="genericFilter"
+      (update)="onFilter($event)"></aui-table-input-filter>
+    <ng-container *ngFor="let filter of visibleFilters" [ngSwitch]="filter.type">
+      <ng-container *ngSwitchCase="'select'">
+        <aui-table-select-filter class="aui-smart-table-filter" [filter]="filter" (update)="onFilter($event)">
+        </aui-table-select-filter>
+      </ng-container>
+      <ng-container *ngSwitchCase="'input'">
+        <aui-table-input-filter class="aui-smart-table-filter" [filter]="filter" (update)="onFilter($event)">
+        </aui-table-input-filter>
+      </ng-container>
+      <ng-container *ngSwitchCase="'datepicker'">
+        <aui-table-datepicker-filter class="aui-smart-table-filter" [filter]="filter" (update)="onFilter($event)">
+        </aui-table-datepicker-filter>
+      </ng-container>
+    </ng-container>
+  </div>
+  <div class="aui-smart-table-filters u-margin-top-xs">
+    <div class="aui-smart-table-filter a-input" *ngIf="optionalFilters.length">
+      <div class="a-input__wrapper">
+        <button class="a-button-outline has-icon-left" (click)="toggleOptionalFilters()">
+          <span class="fa fa-filter"></span> {{ options.translations?.moreFilters || 'Meer filters'}}
+        </button>
+      </div>
+    </div>
+    <div class="aui-smart-table-filter a-input" *ngIf="totalResults > 0">
+      <div class="a-input__wrapper">
+        <button class="a-button-outline a-button-export has-icon-left" (click)="exportToExcel()">
+          <span class="fa fa-file-excel-o"></span> <span class="a-export-label">{{ options.translations?.export || 'Export'}}</span>
+        </button>
+      </div>
+    </div>
+    <div class="aui-smart-table-filter a-input" *ngIf="totalResults > 0">
+      <div class="a-input__wrapper">
+        <div auiFlyout size="small" align="left">
+          <button class="a-button-outline has-icon" title="Open columnselector flyout" auiFlyoutAction><span
```

Comment:
title zou vertaalbaar moeten zijn met nederlandstalige fallback

Reply:
Should be fixed now